.github/workflows/test.yaml

```diff
@@ -2,6 +2,9 @@ name: "Rspec for Rails Apps"
 on:
   workflow_dispatch:
     inputs:
+      base:
```

Comment:
can we use *Target for all of these? that way there is only one variable (baseTarget in this case) that works the same across linters, tests and build. It accomplishes the same thing but with one fewer concepts

Reply:
@orangewolf Let me know what you think of my changes, I appreciate the suggestions to simplify!